src/main/java/com/mixpanel/android/mpmetrics/AnalyticsMessages.java

```diff
@@ -6,6 +6,7 @@
 import android.os.HandlerThread;
 import android.os.Looper;
 import android.os.Message;
+import android.os.SystemClock;
```

Comment:
This is better than how we do things elsewhere- we use currentTimeMillis() inappropriately in other code! Terrific for finding this-I'm going to file a new bug to fix it elsewhere!